temba/locations/migrations/0001_initial.py

```diff
@@ -32,14 +38,22 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='BoundaryAlias',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
-                ('is_active', models.BooleanField(default=True, help_text=b'Whether this item is active, use this instead of deleting')),
-                ('created_on', models.DateTimeField(help_text=b'When this item was originally created', auto_now_add=True)),
-                ('modified_on', models.DateTimeField(help_text=b'When this item was last modified', auto_now=True)),
-                ('name', models.CharField(help_text=b'The name for our alias', max_length=128)),
-                ('boundary', models.ForeignKey(related_name=b'aliases', to='locations.AdminBoundary', help_text=b'The admin boundary this alias applies to')),
-                ('created_by', models.ForeignKey(related_name=b'locations_boundaryalias_creations', to=settings.AUTH_USER_MODEL, help_text=b'The user which originally created this item')),
-                ('modified_by', models.ForeignKey(related_name=b'locations_boundaryalias_modifications', to=settings.AUTH_USER_MODEL, help_text=b'The user which last modified this item')),
+                ('id', models.AutoField(verbose_name='ID',
+                                        serialize=False, auto_created=True, primary_key=True)),
+                ('is_active', models.BooleanField(
+                    default=True, help_text=b'Whether this item is active, use this instead of deleting')),
+                ('created_on', models.DateTimeField(
+                    help_text=b'When this item was originally created', auto_now_add=True)),
+                ('modified_on', models.DateTimeField(
+                    help_text=b'When this item was last modified', auto_now=True)),
+                ('name', models.CharField(
+                    help_text=b'The name for our alias', max_length=128)),
+                ('boundary', models.ForeignKey(related_name=b'aliases', to='locations.AdminBoundary',
+                                               help_text=b'The admin boundary this alias applies to')),
+                ('created_by', models.ForeignKey(related_name=b'locations_boundaryalias_creations',
+                                                 to=settings.AUTH_USER_MODEL, help_text=b'The user which originally created this item')),
+                ('modified_by', models.ForeignKey(related_name=b'locations_boundaryalias_modifications',
+                                                  to=settings.AUTH_USER_MODEL, help_text=b'The user which last modified this item')),
```

Comment:
plz remove these, no reason to reformat old migrations, just makes the diffs confusing